src/core/public/chrome/ui/header/header_breadcrumbs.tsx

```diff
@@ -67,7 +69,7 @@ export function HeaderBreadcrumbs({ appTitle$, breadcrumbs$ }: Props) {
       breadcrumbs={crumbs}
       max={10}
       data-test-subj="breadcrumbs"
-      className="osdHeaderBreadcrumbs"
```

Comment:
I can understand for the shape but looking at the original implementation could the theming being retrieved and applied by the shared ui dep package and applied as a prop to the existing bread crumb. That way the shape is handled in our repo but we rely on the theme from OUI?

Reply:
We have not made any changes to OUI yet! this whole css/style issue will be part of OUI in future! we will have separate ticket for that

Reply:
Yeah sorry about the confusion. What I mean is how come the bread crumb class that we've added has it's own dark mode css instead of pulling the theme from EUI via the shared ui dep package? Seems like that is the standard within the repo.

Reply:
I thought these these dark mode changes will be part of OUI theming in future but at the moment they are not. so we have diffs css! I will confirm with ux. team once! Is it right @kgcreative

Reply:
I'm probably not the best person to talk about CSS implementation details. I would follow whatever guidance we have from a system perspective. The long term plan is to fold these into the updated OUI theme, but we're a long way from that.